tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/Util.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package cloudflow.operator.action
+
+import akka.datap.crd.App
+import cloudflow.operator.action.Common.jsonToConfig
+import io.fabric8.kubernetes.api.model.{ OwnerReference, OwnerReferenceBuilder }
+
+// TODO: separate the concerns and model better this class
+object Util {
+
+  val PrometheusAgentKey = "prometheus"
+
+  def getOwnerReference(name: String, uid: String): OwnerReference = {
+    new OwnerReferenceBuilder()
+      .withController(true)
+      .withBlockOwnerDeletion(true)
+      .withApiVersion(App.ApiVersion)
+      .withKind(App.Kind)
+      .withName(name)
+      .withUid(uid)
+      .build()
+  }
+
+  // TODO: we should keep a single source of truth for model
+  import cloudflow.blueprint.VolumeMountDescriptor
+  import cloudflow.blueprint.deployment._
+  def toBlueprint(deployment: App.Deployment): StreamletDeployment = {
```

Comment:
Why is this not a method on `App.Deployment`?

Reply:
Or in `StreamletDeployment` object, as `fromBlueprint`?

Reply:
moved as a private method to the `Runner`